Userland/Applications/CharacterMap/CharacterSearchWidget.cpp

```diff
@@ -94,5 +95,11 @@ void CharacterSearchWidget::search()
         builder.append_code_point(code_point);
 
         model.add_result({ code_point, builder.to_deprecated_string(), move(display_name) });
+
+        // Stop when we reach 250 results.
+        // This is already too many for the search to be useful, and means we don't spend forever recalculating the column size.
+        if (model.row_count({}) >= 250)
```

Comment:
The commit message still says 500 :sweat_smile:

Reply:
:rofl: I specifically remember thinking when I wrote that: "It's probably a bad idea to put this in the commit message because I'm likely to tweak it... nah, I'll definitely remember to change it."